tensorboard/plugins/graph/keras_util.py

```diff
@@ -92,89 +121,171 @@ def _is_model(layer):
   return layer.get('config').get('layers') is not None
 
 
-def _norm_to_list_of_layers(maybe_layers):
-  """Normalizes to a list of layers.
+def _get_inbound_nodes(tensor_key_with_args_array):
+  """Return normalized list of TensorKeyWithArgs.
+
+  Keras model serialization records input to a layer in a property,
+  `tensor_key_with_args_array`, and it can be of below forms:
+  - [['name_1', 1, 0], ['name_2', 1, 1]]: two invocations witn single input.
+  - [
+      [['name_1', 0, 0], ['name_2', 1, 0]],
+      [['name_3', 0, 0], ['name_4', 1, 0]]]: two invocations with multiple
+    inputs. Multiple inputs are only applicable to limited keras layers like
+    keras.layers.add, keras.layers.concat, and etc...
+
+  Note that there cannot be fixture of types as a layer that takes multiple
```

Comment:
What is a “fixture of types”, and what does it have to do with the size
of the input? Like, the input data shape and batch size? I don’t
understand this proviso.